cmd/main.go

```diff
@@ -355,9 +359,13 @@ func main() {
 				"assisted-service-baseiso-helper",
 				log.WithField("pkg", "baseISOUploadLeader"))
 
-			failOnError(baseISOUploadLeader.RunWithLeader(context.Background(), func() error {
-				return objectHandler.UploadBootFiles(context.Background())
-			}), "Failed uploading boot files")
+			for version := range openshiftVersionsMap {
+				go func(version string) {
+					failOnError(baseISOUploadLeader.RunWithLeader(context.Background(), func() error {
```

Comment:
It would be nice to have a leader per version and parallelize the uploads, but that's for another PR